model/gen-ai/spans.yaml

```diff
@@ -305,6 +315,14 @@ groups:
       - ref: gen_ai.tool.type
         requirement_level:
           recommended: if available
+
+      # New attributes added to execute_tool span
+      - ref: gen_ai.tool.call.arguments
+        requirement_level:
+          conditionally_required: if available
+      - ref: gen_ai.tool.call.result
```

Comment:
I think these should be omitted if the user has not opted in to including content, they should be treated similarly to input/output messages.